### PR TITLE
Created a cpp file in Bestpractice folder.

### PR DESCRIPTION
**Pull request by Abhishek07-0302**

### CODE REVIEW

File: 2021/C-Plus-Plus/BestPractice/hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include<bits/stdc++.h>
```
Comment (Vitaljaz) [inline 2021/C-Plus-Plus/BestPractice/hello_world.cpp:1]: better: `iostream`

File: 2021/C-Plus-Plus/BestPractice/hello_world.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+using namespace std;
+int main(){
```
Comment (Vitaljaz) [inline 2021/C-Plus-Plus/BestPractice/hello_world.cpp:3]: need correct white spaces for good reading

**Review by Vitaljaz — CHANGES_REQUESTED** [review]
Good, but need small refactoring

File: 2021/C-Plus-Plus/BestPractice/hello_world.cpp [hunk 3]
```diff
@@ -1,6 +1,12 @@
-#include<bits/stdc++.h>
+#include<iostream>
```
Comment (Vitaljaz) [inline 2021/C-Plus-Plus/BestPractice/hello_world.cpp:1]: better: `#include <iostream>` with space

File: 2021/C-Plus-Plus/BestPractice/hello_world.cpp [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+
 using namespace std;
-int main(){
-  cout<<"Hello World";
-return 0;
+
+int main()
```
Comment (Vitaljaz) [inline 2021/C-Plus-Plus/BestPractice/hello_world.cpp:5]: just:
```
int main() {
    std::cout << "Hello World!\n";
    return 0;
}
```

**Review by Vitaljaz — CHANGES_REQUESTED** [review]

**Comment by Vitaljaz** [comment]
@Abhishek07-0302 nice, you need star this repo

**Comment by Abhishek07-0302** [comment]
> @Abhishek07-0302 nice, you need star this repo

Done.

**Comment by Abhishek07-0302** [comment]
Please add hacktoberfest accepted tag. If possible.

**Pull request merged**